### ISSUE TITLE
apply for rbind.io-subdomain

### ISSUE THREAD

**Issue by Chris-Engelhardt** [rbind/support#291]
## Netlify website address

https://chris-engelhardt.netlify.com/

## Preferred rbind.io subdomain

chris-engelhardt.rbind.io

### Agreement

- [x] By submitting this request, I promise I will at least write one blog post or create one web page on my website after I get the rbind.io subdomain.

**Comment by road2stat** [comment]
@Chris-Engelhardt -looking good, you're welcome!

To further redirect HTTP to HTTPS automatically for Netlify websites, you may also find the last two sections of this post helpful: https://yihui.name/en/2017/11/301-redirect/.

**Comment by Chris-Engelhardt** [comment]
Yes. I do need to configure https. Thank you for pointing me to this resource!

**Comment by yihui** [comment]
We don't have a strict policy, but I prefer not having an rbind website associated with business. I recommend that you register a top-level domain name by yourself. It is really cheap. Thanks for your understanding!

**Comment by Chris-Engelhardt** [comment]
@yihui I totally understand. Thanks for considering!
